### PR TITLE
Aid jupyter book

### PR DESCRIPTION
**Pull request by jethayden**
Hello, this includes the jupyter book called 'aidbook'.
When you pull these changes and can see the aidbook folder in your jupyter lab, open terminal and run "jupyter-book build aidbook"
You'll get a file link in terminal that takes you to the book! We can make further edits where necessary but test this works

### CODE REVIEW

File: aidbook/PFA logo2.png
Comment (numail) [inline aidbook/PFA logo2.png:1]: really like the logo!

File: aidbook/analysis-numail.Rmd [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+# Print or inspect pos
+print(pos)
+
+# Draw the graph with node sizes based on aid amount
+plt.figure(figsize=(12, 8))
+nx.draw(G, pos, with_labels=True, node_size=[total_aid_per_country.get(country, 1) for country in G.nodes], node_color='skyblue', font_size=8, font_color='black', font_weight='bold', edge_color='gray', linewidths=0.5, arrowsize=10)
+plt.title('Aid Inflow Diagram with Node Size based on Aid Amount (Amount > 0)')
+plt.show()
+
+```
```
Comment (numail) [inline aidbook/analysis-numail.Rmd:309]: this doesn't include all of my code, but hopefully after I pull request it should

File: aidbook/_config.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+repository:
+  url: https://github.com/executablebooks/jupyter-book  # Online location of your book
+  path_to_book: docs  # Optional path to your book, relative to the repository root
+  branch: master  # Which branch of the repository should be used when creating links (optional)
+
+# Add GitHub buttons to your book
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
```
Comment (numail) [inline aidbook/_config.yml:32]: this looks good!

**Review by numail — APPROVED** [review]

File: aidbook/analysis-numail.Rmd (hunk 1 above)
Comment (jethayden) [reply aidbook/analysis-numail.Rmd:309]: gotcha, once you PR ill add it

File: aidbook/PFA logo2.png
Comment (jethayden) [reply aidbook/PFA logo2.png:1]: thank you!! canva came through.

**Pull request merged**